src/emulator/functionsEmulator.ts

```diff
@@ -750,6 +771,36 @@ export class FunctionsEmulator implements EmulatorInstance {
     }
   }
 
+  // Currently only cleans up eventarc and firealerts triggers
+  async removeTriggers(toRemove: string[]) {
+    for (const triggerKey of toRemove) {
+      const definition = this.triggers[triggerKey].def;
+      const service = getFunctionService(definition);
+      const key = this.getTriggerKey(definition);
+
+      switch (service) {
```

Comment:
Note to self/Brian/Daniel - we might need to add PubSub triggers here in to fix https://github.com/firebase/firebase-tools/issues/6585